modules/manage/pages/cluster-maintenance/cluster-diagnostics.adoc

```diff
@@ -11,7 +11,24 @@ When anomalous behavior arises in a cluster and you're trying to figure out whet
 
 Self-test runs a set of benchmarks to determine the maximum performance of a machine's disks and network connections. For disks, it runs throughput and latency tests by performing concurrent sequential operations. For networks, it selects unique pairs of Redpanda nodes as client/server pairs, then it runs throughput tests between them. Self-test runs each benchmark for a configurable duration, and it returns IOPS, throughput, and latency metrics.
 
-=== Self-test command examples
+== Cloud storage tests
+
+If you use xref:manage:tiered-storage.adoc[Tiered Storage], run self-test to verify that you have configured your cloud storage accounts correctly.
+
+Self-test performs the following tests to validate cloud storage configuration:
```

Comment:
```suggestion
Self-test performs the following tests to validate object storage configuration:
```

Reply:
@kbatuigas I think all instances of "cloud storage" here should be "object storage", similar to our Tiered Storage doc page. And I think "bucket" should be "bucket or container".

Reply:
@kbatuigas Although we have this comment, agree with bucket/container.

Reply:
@kbatuigas use the same text that we have here  (or single-source)
https://deploy-preview-604--redpanda-docs-preview.netlify.app/24.2/reference/rpk/rpk-cluster/rpk-cluster-self-test-start/